pkg/codefresh/codefresh.go

```diff
@@ -135,7 +135,13 @@ func (c *codefresh) AppProxy(ctx context.Context, runtime string, insecure bool)
 		return nil, fmt.Errorf("failed to create app-proxy client for runtime %s: %w", runtime, err)
 	}
 
-	if rt.IngressHost == nil || *rt.IngressHost == "" {
+	host := "N/A"
```

Comment:
```suggestion
	var host string
```
since it will never be "n/a" you should just intialize as empty string.